server/src/schemas/holdsSchema.ts

```diff
@@ -1,38 +1,26 @@
 import { gql } from "apollo-server-express";
 
-export interface Hold {
-  id: number;
-  description: String;
-}
-
 export const HoldsTypeDefs = gql`
-  enum HoldType {
-    INCOMPLETE_TRAINING
-    SAFETY_VIOLATION
-    BALANCE_DUE
-  }
-
   type Hold {
     id: ID!
-    type: HoldType!
+    userID: User!
```

Comment:
```suggestion
    user: User!
```

Reply:
Do you disagree with this suggestion? The conversation was marked as resolved but the change wasn't implemented.